cclib/scripts/ccget.py

```diff
@@ -159,19 +159,28 @@ def ccget() -> None:
             name = f"{', '.join(filename[:-1])} and {filename[-1]}"
         else:
             name = filename
+        
+        # Set custom handles so we can actually change log-levels.
+        handler = logging.StreamHandler(sys.stderr)
+        handler.setFormatter(logging.Formatter("[%(name)s %(levelname)s] %(message)s"))
+        logging.getLogger("cclib").addHandler(handler)
 
         # The keyword dictionary are not used so much. but could be useful for
         # passing options downstream. For example, we might use --future for
         # triggering experimental or alternative behavior (as with optdone).
         kwargs = {}
         if verbose:
-            kwargs['verbose'] = True
             kwargs['loglevel'] = logging.INFO
+            logging.getLogger("cclib").setLevel(logging.INFO)
+        
         else:
-            kwargs['verbose'] = False
+            # TODO: Are we sure we want to ignore warnings by default?
             kwargs['loglevel'] = logging.ERROR
+            logging.getLogger("cclib").setLevel(logging.WARNING)
```

Comment:
These two lines look conflicting...is that "error" for the global logger and "warning" for the logfile instance logger?

I would agree with your comment that both should be warning.

Reply:
Yes exactly, but I'll update so both are WARNING so it's less weird :). This will make some parsers appear more noisy (Orca in particular emits quite a few warnings I think), but to me this is the better behaviour so a non-issue.